docs/platform_ccl.rst

```diff
@@ -0,0 +1,297 @@
+	  "connection_map":
+	  {
+	    "0" : [0, 1],
+	    "1" : [2, 3, 4],
+	    "2" : [5, 6]
+	  }
+	},
+
+The number of these waveform generators is specified by the ``count`` field. In the ``connection_map`` it is specified which wafeform generator is connected to which qubits. For instance, Line 6 specifies that ``qwg 0`` is connected to qubits 0 and 1. This is based on ``AWG-8 1, channel 0`` entry in 4th row in Table :numref:`table_ccl_connections` This information is utilized by the scheduler to perform resource-constraint aware scheduling of instructions.
+
+.. note::
+	By providing an empty list for a qwg will result in not applying any qwg constraint during scheduling.
+
+Available measurement/readout units are specified in ``meas_units`` section, as shown below.
+
+.. code-block::
+   :linenos:
+
+	"meas_units" :
+	{
+	  "count": 2,
+	  "connection_map":
+	  {
+	    "0" : [0, 2, 3, 5, 6],
+	    "1" : [1, 4]
+	  }
+	},
+
+
+.. _fig_qubit_numbering_ccl:
+
+.. figure:: ./qubit_number.png
+    :width: 800px
+    :align: center
+    :alt: Qubit numbering and connectivity of qubits in CC-Light Platform
+    :figclass: align-center
+
+    Qubit numbering and connectivity in CC-Light Platform.
+
+
+For the details below, it will be convinient to consider the 
+Figure :numref:`fig_qubit_numbering_ccl`, which shows qubit and edge numbering in CC-Light platform.
+
+
+Edges and the constraints imposed by these edges are specified in ``edges`` section.
+``count`` at Line 3 specifies the number of edges between qubits in the platform.
+``connection_map`` specifies connections. For example, on Line 6, Edge 0 implies a
+constraint on Edge 2 and Edge 10. This means, if Edge 0 is reserved for an operation,
+an operation on Edge 2 and Edge 10 will not be scheduled, until operation on Edge 0
+is complete.
+
+.. code-block::
+   :linenos:
+
+	"edges":
+	{
+	  "count": 16,
+	  "connection_map":
+	  {
+	    "0": [2, 10], 
+	    "1": [3, 11],
+	    "2": [0, 8],
+	    "3": [1, 9],
+	    "4": [6, 14],
+	    "5": [7, 15],
+	    "6": [4, 12],
+	    "7": [5, 13],
+	    "8": [2, 10],
+	    "9": [3, 11],
+	    "10": [0, 8],
+	    "11": [1, 9],
+	    "12": [6, 14],
+	    "13": [7, 15],
+	    "14": [4, 12],
+	    "15": [5, 13]
+	  }
+	},
+
+
+Detunning constraints are specified ``detuned_qubits``. A two-qubit flux gate
+lowers the frequency of its source qubit to get near the frequency of its target
+qubit.  Any two qubits which have near frequencies execute a two-qubit flux gate.
+To prevent any neighbor qubit of the source qubit that has the same frequency as
+the target qubit to interact as well, those neighbors must have their frequency
+detuned (lowered out of the way).  A detuned qubit cannot execute a single-qubit
+rotation (an instruction of 'mw' type).  An edge is a pair of qubits which can
+execute a two-qubit flux gate.  There are 'count' qubits. For each edge it is
+described, when executing a two-qubit gate for it, which set of qubits it detunes.
```

Comment:
`which set of qubits it detunes.` does this mean that OpenQL can automatically insert detuning pulse (flux) on a third qubit when a user specifies a two-qubit operation on two qubits (in case the third qubit is of the same frequency of the first)?
If so this should be more clearly described and put in a WARNING/NOTE box, since this feature may explicitly change the operations a user programs in his code (by adding additional ones, which may not be desired)...